### PR TITLE
Empty boxes configuration docs

### PR DESCRIPTION
**Pull request by kunicmarko20**
I am targeting this branch, because this is docs.

## Subject

If there is no documentation, it's like the feature does not exist :smile:

### CODE REVIEW

File: docs/reference/search.rst [hunk 1]
```diff
@@ -85,3 +85,47 @@ The current implementation can be expensive if you have a lot of entities as the
...
+.. figure:: ../images/empty_boxes_show.png
+    :align: center
+    :alt: Custom view
+    :width: 700px
+
+We can fade out the boxes that have no results with:
+
+.. code-block:: yaml
+
+    # app/config/config.yml
```
Comment (jordisala1991) [inline docs/reference/search.rst]: shouldn't this be something like sonata_config, or it is okay like that?

File: docs/reference/search.rst [hunk 2]
```diff
@@ -85,3 +85,47 @@ The current implementation can be expensive if you have a lot of entities as the
...
+    sonata_admin:
+        global_search:
+            empty_boxes: hide
+
+and it looks like:
+
+.. figure:: ../images/empty_boxes_hide.png
+    :align: center
+    :alt: Custom view
+            :width: 700px
```
Comment (jordisala1991) [inline docs/reference/search.rst]: wrong indent

File: docs/reference/search.rst [hunk 3]
```diff
@@ -85,3 +85,47 @@ The current implementation can be expensive if you have a lot of entities as the
 .. note::
 
     There is a work in progress to use an async JavaScript solution to better load data from the database.
+
+Customize visibility of empty result boxes
+------------------------------------------
+
+By default all the admin boxes are show in search results and it looks like:
```
Comment (greg0ire) [inline docs/reference/search.rst]: - Shown
- Like this

File: docs/reference/search.rst [hunk 4]
```diff
@@ -85,3 +85,47 @@ The current implementation can be expensive if you have a lot of entities as the
...
+
+.. code-block:: yaml
+
+    # app/config/config.yml
+
+    sonata_admin:
+        global_search:
+            empty_boxes: fade
+
+and it looks like:
```
Comment (greg0ire) [inline docs/reference/search.rst]: Like this

File: docs/reference/search.rst (hunk 1 above)
Comment (jordisala1991) [reply docs/reference/search.rst]: Well, if `config/packages/sonata_admin.yaml` is more correct, then yes IMO.

**Comment by greg0ire** [comment]
How much do the images weight? Should they maybe be resized to fit the design of the current docs?

File: docs/reference/search.rst (hunk 1 above)
Comment (kunicmarko20) [reply docs/reference/search.rst]: should I edit the whole document then and fix it everywhere? `config/packages/sonata_admin.yaml` is correct if you are using flex.

**Comment by kunicmarko20** [comment]
Around 60k per image. I can resize them if needed?

**Comment by jordisala1991** [comment]
https://imageoptim.com/mac

**Comment by kunicmarko20** [comment]
Should I do it for all images? Maybe in a new PR

**Comment by greg0ire** [comment]
Make it in this PR please. Every modification you make to an image gets stored into git. Apparently, images are displayed with a width of 700px: https://sonata-project.org/bundles/admin/3-x/doc/getting_started/creating_an_admin.html

**Comment by greg0ire** [comment]
I think that I'd crop the image to leave out the right part so that it looks the normal size even when constrained to 700px

**Comment by greg0ire** [comment]
Here is how it could look:

empty_boxes_fade

**Comment by greg0ire** [comment]
A fresh clone weights 41M so it's a bit late to worry about 60K I'm afraid :sweat_smile: , so let's just focus on the legibility issue and tell me if you prefer the cropped or full screen version

**Comment by kunicmarko20** [comment]
We can go with cropped if you want, same for me. (will you crop all three? :smile:)

**Comment by greg0ire** [comment]
empty_boxes_show
empty_boxes_hide

**Comment by greg0ire** [comment]
Thanks @kunicmarko20 !

**Pull request merged**